SocialFilter/sections/specific.txt

```diff
@@ -1912,6 +1912,8 @@ stylehaus.jp##.f-section-sns
 trends.google.*##share[ng-if="shareConfig"]
 cosme.net##.act-sbmlist
 cosme.net##.social-item-block
+s.cosme.net###sns-shere
+s.cosme.net###scl-lst
```

Comment:
Is it = necessary to additionally hide the `#scl-lst`?

Reply:
I checked that, and yes. It blocks blank spaces of share buttons.